src/trie-prefix-tree.d.ts

```diff
@@ -0,0 +1,70 @@
+// Type definitions for trie-prefix-tree 1.5
+// Project: https://github.com/lyndseybrowning/trie-prefix#readme
+// Definitions by: James Lismore <https://github.com/jlismore>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+declare module 'trie-prefix-tree'

```

Comment:
Add a blank line and a comment explaining why this file is necessary.

Reply:
I think your cherry-picked commit includes this